sycl/test-e2e/Graph/ValidUsage/linear_graph_l0_copy.cpp

```diff
@@ -0,0 +1,103 @@
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+// Extra run to check for leaks in Level Zero using UR_L0_LEAKS_DEBUG
+// RUN: env SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=0 %{l0_leak_check} %{run} %t.out 2>&1 | FileCheck %s --implicit-check-not=LEAK
+// Extra run to check for immediate-command-list in Level Zero
+// RUN: env SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1 %{l0_leak_check} %{run} %t.out 2>&1 | FileCheck %s --implicit-check-not=LEAK
+//
+// REQUIRES: level_zero
```

Comment:
Does this test need to require L0? The test should still work on all backends even if the interaction that it is intended to stress is L0.

Reply:
Hmm, I suppose not! I've removed it now.